front/src/modules/users/components/Avatar.tsx

```diff
@@ -79,13 +84,19 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
         return '12px';
     }
   }};
+
+  &:hover {
+    border-color: ${({ onClick }) =>
+      onClick ? 'rgba(0, 0, 0, 0.1)' : 'transparent'};
```

Comment:
The issue mentions a "transparent lighter" color, so here I assume we would like to use `theme.background.transparent.lighter`. Is that correct @Bonapara ?

Reply:
Correct for the color! The border weight should be 4px

Reply:
Thanks @thaisguigon!